src/wallet/wallet.cpp

```diff
@@ -1507,7 +1507,7 @@ bool DummySignInput(const SigningProvider& provider, CTxIn &tx_in, const CTxOut
     return true;
 }
 
-static void FillInputToWeight(CTxIn& txin, int64_t target_weight)
```

Comment:
nit: Seems like changing `FillInputToWeight` to not be static and adding its unit tests were squashed into 8a1ea6a050 but should have been squashed into 9191748088 instead?

Reply:
Oops, fixed.